tests/testthat/test-drive_ls.R

```diff
@@ -0,0 +1,34 @@
+context("List files")
+
+nm_ <- nm_fun("-TEST-drive-ls")
+
+run <- FALSE
+clean <- FALSE
+if (run) {
+  ## make sure directory is clean
+  if (clean) {
+    del_pths <- c(nm_("foo"), nm_("this-should-not-exist"))
+    del <- purrr::map(del_pths, drive_delete, verbose = FALSE)
```

Comment:
I see. I'm not sure `drive_path()` really can/should take > 1 path. Or maybe it can/should? What do you think? Since we don't include path information, it would seem weird to get files back with no info on which of the input paths they matched.

But when we're using `drive_path()` internally, that seems less of a problem. The mapping could happen inside `drive_delete()`. But then that would apply in many places and we should solve with a function.